### ISSUE TITLE
does not work on mac

### ISSUE THREAD

**Issue by idefixcert** [amuttsch/go-idasen#1]
Any Idea how to port this to mac?

Because I get
AdapterExists: dial unix /opt/local/var/run/dbus/system_bus_socket: connect: no such file or directory

**Comment by amuttsch** [comment]
I don't think this will work on mac as I use https://github.com/muka/go-bluetooth for bluetooth connection which relies on dbus and bluez which is only available on linux.

You could try the python implementation if it works on mac: https://github.com/newAM/idasen

If you are interested in creating a mac version of this the first step would be finding a go bluetooth library that works on mac and try implementing it.
